### DISCUSSION TITLE
You.com Api request

### DISCUSSION THREAD

**Discussion by Yorddyyorddy** [acheong08/EdgeGPT#552] [Show and tell]
what do you think about  you.com api

 @acheong08

**Comment by acheong08** [comment]
Cloudflare on "I'm under attack" mode. Not gonna try

**Comment by Yorddyyorddy** [comment]
that's why you needed ,but it could be easily bypassed by TLs I think

**Reply by acheong08** [reply]
Nope. They seems to be showing captcha to **all** visitors. Solving this is more in the territory of @linweiyuan who is amazing at browser automation

**Reply by Yorddyyorddy** [reply]
but sir I haven't encountered anything like that maybe it depends on your ip

 well that guy kick for us ,if not selenium

**Comment by Yorddyyorddy** [comment]
😫I'm bored with google batchexecute it worse than cloudflare....can you help me how to figure out how f.req=blabla request data work ....the json string which contains array Xsrf ....etc

**Reply by acheong08** [reply]
https://github.com/acheong08/Bard

**Reply by acheong08** [reply]
I'm just parsing as array manually for now

**Reply by Yorddyyorddy** [reply]
good example but actually not the same with which the domain  I'm trying 

but someone has to think of seriously about Google web batch
